woof-code/rootfs-skeleton/usr/share/doc/gtkdialog.htm

```diff
@@ -8,13 +8,16 @@
 pre {
   color: crimson;
   background-color: #f1f1f1;
```

Comment:
Maybe add a link to https://github.com/puppylinux-woof-CE/gtkdialog/wiki?

Reply:
https://github.com/lakshayrohila/woof-CE/blob/b52f113a7384258889f6d34fbf13ec3dbb98dc98/woof-code/rootfs-skeleton/usr/share/doc/gtkdialog.htm#L65 does this.